app/src/main/res/layout/activity_calendar.xml

```diff
@@ -6,6 +6,2386 @@
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/two1"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/three1"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/four1"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/five1"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/six1"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/seven1"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                    </TableRow>
+                    <TableRow>
+                        <TextView
+                            android:text="8:15"
+                            android:background="@color/white"
+                            android:textColor="@color/mainBurgundy"
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/one2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/two2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/three2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/four2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/five2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/six2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
+                        <TextView android:id="@+id/seven2"
+
+                            android:textAlignment="center"
+                            android:gravity="center"></TextView>
```

Comment:
Same as above comment, why do we skip a line between id and textAlignment? needs to be removed and closing tag needs to be on a new line.

Reply:
It was caused by find and replace, and I failed to notice it. Will fix it.